lib/ship_quiet_logistics/documents/shipment_order.rb

```diff
@@ -94,6 +94,14 @@ def date_stamp
       def order_type
         'SO'
       end
+
+      def country_name(address)
+        if address.respond_to?(:country_name)
+         address.country_name
+        elsif address.country.present?
+          address.country.name
+        end
```

Comment:
Is `nil` an okay result, if `address` neither responds to `country_name` or `country`?

Reply:
I don't know! Maybe ?
Would you raise an error for this ?